modules/cart/tests/src/Functional/CartTest.php

```diff
@@ -41,7 +43,6 @@ class CartTest extends OrderBrowserTestBase {
   protected function getAdministratorPermissions() {
     return array_merge([
       'administer products',
-      'access cart',
```

Comment:
The permissions 'access cart' and 'access checkout' are also in /commerce/modules/checkout/tests/src/Functional/CheckoutOrderTest.php
~line 68

Can we remove them there to in this PR?